resources/clus_multi.csv

```diff
@@ -0,0 +1,900 @@
+0,0,0,0,0,0,0,0,0,0,0,0,0,0,0,0,0,0,0,0,1,1,1,1,1,1,1,1,1,1,1,1,1,1,1,1,1,1,1,1
```

Comment:
I propose to remove all the .csv files from the github repo, they are pretty useless as of now